chord_metadata_service/phenopackets/api_views.py

```diff
@@ -252,7 +252,8 @@ def phenopackets_overview(_request):
     get:
     Overview of all Phenopackets in the database
     """
-    phenopackets = m.Phenopacket.objects.all()
+    phenopackets = m.Phenopacket.objects.all().prefetch_related(*PHENOPACKET_PREFETCH).select_related(
+        *PHENOPACKET_SELECT_REL)
```

Comment:
`PHENOPACKET_SELECT_REL` doesn't seem to be initialized anywhere and so this is why the Travis build is failing.